docs/docs/using-web-fonts.md

```diff
@@ -18,39 +18,30 @@ Some examples of web font services include [Google Fonts](https://fonts.google.c
 
 ### Using Google Fonts
 
-The fastest way to get started using Google Fonts is by choosing a font from [the typefaces project](https://github.com/KyleAMathews/typefaces). This example shows how you can add Open Sans to your project.
+The fastest way to get started using Google Fonts is by choosing a font from [Fontsource](https://github.com/fontsource/fontsource).
```

Comment:
In the future, I'm planning on removing either this doc or the Recipe doc, to reduce the redundancy. For now, I would update this section with the same edits I suggested above.